mmdeploy/backend/openvino/onnx2openvino.py

```diff
@@ -55,30 +56,33 @@ def get_output_model_file(onnx_path: str, work_dir: str) -> str:
     return model_xml
 
 
-def onnx2openvino(input_info: Dict[str, Union[List[int], torch.Size]],
-                  output_names: List[str],
-                  onnx_path: str,
-                  work_dir: str,
-                  mo_options: Optional[ModelOptimizerOptions] = None):
+def from_onnx(onnx_model: Union[str, onnx.ModelProto],
+              output_file_prefix: str,
+              input_info: Dict[str, Sequence[int]],
+              output_names: Sequence[str],
+              mo_options: Optional[ModelOptimizerOptions] = None):
     """Convert ONNX to OpenVINO.
 
     Examples:
-        >>> from mmdeploy.backend.openvino.onnx2openvino import onnx2openvino
+        >>> from mmdeploy.apis.openvino import from_onnx
         >>> input_info = {'input': [1,3,800,1344]}
         >>> output_names = ['dets', 'labels']
         >>> onnx_path = 'work_dir/end2end.onnx'
         >>> work_dir = 'work_dir'
-        >>> onnx2openvino(input_info, output_names, onnx_path, work_dir)
+        >>> from_onnx( onnx_path, work_dir, input_info, output_names)
```

Comment:
The example in docstring is supposed to be updated, as the arguments are changed.